modules/economy-system/commands/economy-system.js

```diff
@@ -0,0 +1,378 @@
+const {balance, createleaderboard} = require('../economy-system');
+
+module.exports.beforeSubcommand = async function (interaction) {
+    interaction.str = interaction.client.configurations['economy-system']['strings'];
+    interaction.config = interaction.client.configurations['economy-system']['config'];
+};
+
+module.exports.subcommands = {
+    'work': async function (interaction) {
+        const model = interaction.client.models['economy-system']['cooldown'];
+        const cooldownModel = await model.findOne({
+            where: {
+                userId: interaction.user.id,
+                command: 'work'
+            }
+        });
+        if (cooldownModel) {
+            return interaction.reply({
+                content: interaction.str['cooldown'],
+                ephemeral: true
+            });
+        }
+        const cooldown = await model.create({
+            userId: interaction.user.id,
+            command: 'work'
+        });
+        const moneyToAdd = Math.floor(Math.random() * (interaction.config['maxWorkMoney'] - interaction.config['minWorkMoney'])) + interaction.config['minWorkMoney'];
+        balance(interaction.client, interaction.user.id, 'add', moneyToAdd);
+        interaction.reply({
+            content: interaction.str['workSuccess'].split('%erned%').join(`${moneyToAdd} ${interaction.config['currencySymbol']}`),
+            ephemeral: true
+        });
```

Comment:
Please allow embedType, as a lot of our users love creating beautiful embeds ^^
You could use 
```suggestion
   interaction.reply(embedType(interacction.str['workSuccess'], {'%earned%': `${moneyToAdd} ${interaction.config['currencySymbol']}`}, { ephemeral: true });
```
for example